Tests/XMLDeserIntTestHelper.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Serialization;
+using QCUtilities.Entities;
+
+namespace QCVault.Tests
+{
+    public static class XMLDeserIntTestHelper
+    {
+        public static readonly string IntDestDir = Path.Combine(AppContext.BaseDirectory, "IntegrationTestFolder");
+        public static readonly string SchemaFolder = Path.Combine(AppContext.BaseDirectory, "Utilities", "Entities");
+        public  const string INVALIDXML = "invalid.xml";
+        public  const string EMPTYXML = "empty.xml";
+        public  const string VALIDXML = "valid.xml";
+        public  const string EMPTYTXT = "test.adad";
+        public  const string SCHEMA = "posts.xsd";
+
+        public static string ValidXMLPath => Path.Combine(IntDestDir, VALIDXML);
+
+        public static PostCollection CreateTestCollection()
+        {
+            var posts = new PostCollection
+            {
+                Posts = new Post[2]
```

Comment:
Recommend just rolling this into a big

```
return new PostCollection
{
    Posts = new Post[2]
    {
        new Post()
        {
          /* etc */
        },
        new Post()
        {
          /* etc */
        }
    }
}
```

instead of mucking around with temporary variables.

Alternatively, change PostCollection.Posts into a List<> so you can just .Add() to it.